signbank/query_parameters.py

```diff
@@ -196,6 +196,12 @@ def convert_query_parameters_to_filter(query_parameters):
 
         elif get_key == 'dialect[]':
             query_list.append(Q(dialect__in=get_value))
+
+        elif get_key == 'tags[]':
+            tags = Tag.objects.filter(name__in=get_value)
+            pks_for_glosses_with_tags = [item.object_id for item in TaggedItem.objects.filter(tag_id__in=tags)]
```

Comment:
This could be a single line. Similar to my comment in adminviews.py.

Reply:
> This could be a single line. Similar to my comment in adminviews.py.

This ended up being as it is because it's awkward to get things out of the TaggedItem result.
The objects themselves can be any object, not just glosses.

What confuses me is that the TaggedItem supposedly have tag_id and object_id, but it doesn't need to be an id.
I suspect I wrote it so to force it to evaluate it as specified. :)

Reply:
Hmmm. It seems impossible to create a query that has a '' inside the getlist !!

This may be a remnant from before the field choice model was created, when they used to be "char" fields and could be empty.
It could also be a remnant from when the lists were hard coded and also included empty choices.

I'll remove them!